client/src/index.js

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App';
-
+import { Auth0Provider } from "@auth0/auth0-react";
 
 
 
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-
-  
-  <React.StrictMode>
+  <Auth0Provider
+  domain="dev-iv6sdwq2w07uuaxb.us.auth0.com"
```

Comment:
Keep `clientId` and `domain` in `.env` file.

Reply:
The requested changes will be there in the next to next commit which is on the way.